app/components/moderation-list/template.hbs

```diff
@@ -0,0 +1,57 @@
+<div class="container">
+    <div class="row">
+        <div class="col-md-12 reviews-list">
+            <div class="panel panel-default">
+                <div class="heading">
+                    <div class="btn-toolbar">
+                        <button type="button" class="btn btn-filter" data-target="pending" onclick={{action "buttonPressed" 'pending'}}><a class="fa fa-hourglass-o"></a> {{model.pendingPreprints.meta.pagination.total}} {{t 'moderation_list.pending'}}</button>
+                        <button type="button" class="btn btn-filter" data-target="accepted" onclick={{action "buttonPressed" 'accepted'}}><a class="fa fa-check-circle-o"></a> {{model.acceptedPreprints.meta.pagination.total}} {{t 'moderation_list.accepted'}}</button>
+                        <button type="button" class="btn btn-filter" data-target="rejected" onclick={{action "buttonPressed" 'rejected'}}><a class="fa fa-times-circle-o"></a> {{model.rejectedPreprints.meta.pagination.total}} {{t 'moderation_list.rejected'}}</button>
+                        <div class="dropdown pull-right">
+                            <button class="btn dropdown-toggle" type="button" id="sortBy" data-toggle="dropdown">
+                                {{t 'moderation_list.sort'}}
+                                <span class="fa fa-caret-down"></span>
+                            </button>
+                            <ul class="dropdown-menu dropdown-menu-right" id="sortByOptionList">
+                                <span>
+                                    <button class="btn" onclick={{action "sortSelected" '-date_created'}}><a class='{{if (eq sort '-date_created') 'fa fa-check'}}'></a> {{t 'moderation_list.newest'}}</button>
+                                    <button class="btn" onclick={{action "sortSelected" 'date_created'}}><a class='{{if (eq sort 'date_created') 'fa fa-check'}}'></a> {{t 'moderation_list.oldest'}}</button>
+                                </span>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                {{#if loading}}
+                    <div class="text-center p-v-md" aria-label='Loading data'>
```

Comment:
HTML attributes should be in double quotes, not single. Mostly just for convention but it also makes it a bit easier to distinguish between static and bound attributes.